frontend/.env.example

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=https://code4rocoviz19api-demo.azurewebsites.net/api/v2
```

Comment:
What is the purpose of this file?

Reply:
This is for development on frontend. Every user should copy .env.example in .env and change the file to the desired URL. Maybe I should update the set up part of the readme

Reply:
:+1: